scipy/stats/stats.py

```diff
@@ -944,8 +944,17 @@ def moment(a, moment=1, axis=0):
             # the input was 1D, so return a scalar instead of a rank-0 array
             return np.float64(0.0)
     else:
-        mn = np.expand_dims(np.mean(a, axis), axis)
-        s = np.power((a - mn), moment)
+        # Subtract mean along the axis, compute element-wise square
+        a_zero_mean = a - np.expand_dims(np.mean(a, axis), axis)
+        a_zero_mean_2 = a_zero_mean**2
+        
+        s = a_zero_mean_2.copy()
+        for k in range(1, mom // 2):
```

Comment:
`mom` should be `moment`. Which is the source of the TravisCI failures. 
Generally, it's a good idea to run tests locally, eg `$python runtests.py -s stats`